Source/WebCore/rendering/RenderLayerScrollableArea.cpp

```diff
@@ -1023,6 +1019,13 @@ OverscrollBehavior RenderLayerScrollableArea::verticalOverscrollBehavior() const
     return OverscrollBehavior::Auto;
 }
 
+ScrollbarGutter RenderLayerScrollableArea::scrollbarGutterStyle()  const
+{
+    if (m_layer.renderBox())
```

Comment:
I think it would be better style to put this into a local and use it. Can still do that inside the `if` condition.

Reply:
Apologies I'm not sure what you mean?

Reply:
```
if (auto* renderer = m_layer->renderBox())
    return renderer->style() etc.
```

Reply:
Ah yes that makes sense apologies I was confused because it's not actually renderBox that is called inside of the if statement but there's no reason it can't be. Begs the question why the overscrollBehaviour properties are called the way they are? (scrollbarWidthStyle is the way it is because I copied those)